insightconnect_plugin_runtime/plugin.py

```diff
@@ -219,6 +221,13 @@ def envelope(
                         "data": ex.data,
                     }
 
+                if is_test and not isinstance(ex, ConnectionTestException):
+                    output_message["exception"] = {
+                        "cause": "SDK: Connection test failed.",
```

Comment:
```suggestion
                        "cause": "Plugin connection test failed.",
```
"SDK" isn't too relevant to the user - it's not something they're likely to be familiar with. But, they do know what plugin means.